Stripe/STPPaymentCardTextField.m

```diff
@@ -780,6 +783,23 @@ - (void)updateImageForFieldType:(STPCardFieldType)fieldType {
     }
 }
 
+- (NSString *)defaultCVCPlaceholder {
+    if (self.viewModel.brand == STPCardBrandAmex) {
+        return @"CVV";
+    } else {
+        return @"CVC";
+    }
+}
+
+- (void)updateCVCPlaceholder {
+    if (self.cvcPlaceholder) {
+        self.cvcField.placeholder = self.cvcPlaceholder;
+    } else {
+        self.cvcPlaceholder = [self defaultCVCPlaceholder];
+    }
+    self.cvcField.accessibilityLabel = [self defaultCVCPlaceholder];
```

Comment:
`self.cvcPlaceholder` is a property that we expose that lets you set a custom CVC placeholder – if it's set, we use that, otherwise we use `CVC` or `CVV`.

We should probably localize `CVC` and `MM/YY`, but I think we should leave that out of the scope of this PR – I'll file a task to keep track of it.